model/stacktrace_frame.go

```diff
@@ -97,9 +96,9 @@ func (s *StacktraceFrame) applySourcemap(mapper sourcemap.Mapper, service Servic
 	sourcemapId := s.buildSourcemapId(service)
 	mapping, err := mapper.Apply(sourcemapId, s.Lineno, *s.Colno)
 	if err != nil {
-		logp.Err(fmt.Sprintf("failed to apply sourcemap %s", err.Error()))
-		e, issourcemapError := err.(sourcemap.Error)
-		if !issourcemapError || e.Kind == sourcemap.MapError || e.Kind == sourcemap.KeyError {
+		logp.NewLogger("stacktrace").Errorf("failed to apply sourcemap %s", err.Error())
```

Comment:
what do you think about adding a `var logger = logp.NewLogger("stacktrace")` as a global near the top of the file here instead? similar to how we do in python

Reply:
need to check if it's safe for multi-threaded usage